code-scanning/properties/eslint.properties.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "ESLint",
+    "description": "A tool for identifying and reporting the problems found in ECMAScript/JavaScript code.",
+    "iconName": "eslint",
+    "categories": [
+        "Code Scanning",
+        "javascript",
+        "ecmascript",
```

Comment:
Is there way do detect EcmaScript? https://github.com/github/linguist/blob/master/lib/linguist/languages.yml doesn;t have logic for it?

Reply:
My understanding is that the ECMAScript is spec/standard for scripting languages like JavaScript. I think the ESLint inspects the source code by checking the ECMAScript specification. Should it be added in the categories?